index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <section class="first-sector">
+        <div class="header-comtainer">
+            <div class="header">
+                <div class="my-logo">My Logo</div>
```

Comment:
- [ ] By convention, the logo is a link that redirects the user to the home page, so in order to provide a good user experience, it would be good if you can change the `<div>` tag for an anchor `<a>` one. 🤓 🛠️ 

```suggestion
                <a href="#" class="my-logo">My Logo</a>
```